.github/workflows/client-report-chart-version.yaml

```diff
@@ -0,0 +1,286 @@
+name: Report Chart Version
+
+# This workflow is meant to be called from other repositories' workflows to report a new chart version to Sherlock.
+#
+# The caller repository must have Workload Identity Federation configured to allow impersonation of the
+# "dsp-tools-iap-access@dsp-tools-k8s.iam.gserviceaccount.com" service account; steps 1 and 2 of the documentation:
+# https://docs.google.com/document/d/1bnhDmWQHAMat_Saoa_z28FHwXmGWw6kywjdbKf208h4/edit
+#
+# With that configured, here's how you can call this workflow from whatever workflow currently publishes the chart:
+# ```yaml
+# jobs:
+#
+#   <your-existing-job-id>:
+#     # Output the version from your existing job's tag/bump step, something like this:
+#     outputs:
+#       tag: ${{ steps.tag.outputs.tag }}
+#     steps:
+#       # ... (The rest of your existing job can stay the same)
+#
+#   report-to-sherlock:
+#     uses: broadinstitute/sherlock/.github/workflows/client-report-chart-version.yaml@main
+#     needs: <your-existing-job-id>
+#     with:
+#       new-version: ${{ needs.<your-existing-job-id>.outputs.tag }}
+#       chart-name: '<your-chart-helm-chart-name>'
+#     permissions:
+#       contents: 'read'
+#       id-token: 'write'
+# ```
+
+on:
+  workflow_call:
+
+    secrets:
+      custom-git-token:
```

Comment:
@jyang-broad  Can you describe a scenario where you'd want to use a custom git token?

Reply:
custom-git-token would be used in conjunction w/ custom-git-repo (more below)